src/dotnet-uninstall/Shared/Exceptions/DotNetUninstallExceptions.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System;
+using System.CommandLine;
+
+namespace Microsoft.DotNet.Tools.Uninstall.Shared.Exceptions
+{
+    internal abstract class DotNetUninstallException : Exception
```

Comment:
nit: make them different files. just how we usually do it

Reply:
Done.